force-app/main/default/lwc/edaSettingsNavigation/edaSettingsNavigation.js

```diff
@@ -0,0 +1,39 @@
+import { LightningElement, api } from "lwc";
+import accountModelSettingsTitle from "@salesforce/label/c.stgAccountModelSettingsTitle";
+
+export default class EdaSettingsNavigation extends LightningElement {
+    labelReference = {
+        accountModelSettingsTitle,
+    };
+
+    settingsNavigationViewModel = {
+        navigationSections: [
+            {
+                label: this.labelReference.accountModelSettingsTitle,
+                page: "accountmodelsettings",
+                sectionkey: "accountmodelsettings",
+                isActive: true,
+            },
+            {
+                label: "Contact Information",
+                page: "contactinfosettings",
+                sectionkey: "contactinfosettings",
+            },
+            {
+                label: "Addresses",
+                page: "addresssettings",
+                sectionkey: "addresssettings",
+            },
+        ],
+    };
+
+    @api setActivePage(pageName) {
+        this.settingsNavigationViewModel.navigationSections.forEach((navigationSection) => {
+            if (navigationSection.pageName === pageName) {
+                navigationSection.isActive = true;
+            } else {
+                navigationSection.isActive = undefined;
```

Comment:
Why are we setting isActive as undefined instead of false?

Reply:
This is following truthy/falsy. More importantly, this is a format of defining variables that can be important when dealing with some attributes in lwc, such as disabled on input affordances. In a 'fun' experience with said attributes:

- When a disabled attribute is undefined, the component is considered to not be disabled.
- When disabled === false, the component is disabled. This was the headscratcher moment for me.
- When disabled === true, the component is disabled.
- Thus, when disabled is defined, the component is considered disabled. Fun with Javascript!